File: Algorithms/Bubble Sort/BubbleSort.py
def get_array( ) :
    switch = True
    array = []
    while switch:
        array_element = input("Please enter element of array :  ")
        array.append(array_element) if array_element != "" else 0
        switch = False if array_element == "" else True
    return array

def bubble_sort( array ) :
    result = True
    while result:
        result = False
        size_array = len(array) - 1
        index = 0
        while index < size_array:
            if array[index] > array[index + 1]:
                array[index],array[index + 1] = array[index + 1],array[index]
                result = True
            index += 1
    return array

array = get_array()
print(bubble_sort(array))

File: Algorithms/Bubble Sort/Bubble sort.c
#include <stdio.h>

int bubbleSort(int arr[], int sizeArr);

int main()
{
    int array[] = {2,6,7,8,1,9};
    int sizeArray = sizeof(array) / sizeof(int);
    bubbleSort(array,sizeArray);
    return 0;
}

int bubbleSort(int arr[], int sizeArr)
{
    int result = 1;

    while(result)
    {
        result = 0;

        for(int i = 0; i < sizeArr; i++)
        {
            if(arr[i] > arr[i+1])
            {
                int foo = arr[i];
                arr[i] = arr[i+1];
                arr[i+1] = foo;
                result = 1;
            }
        }
    }
}
